### ISSUE TITLE
[a11y] Audit Popover

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1146]
# Vérification de l'accessibilité du composant : Popover

Afin d'évaluer le niveau d'accessibilité d'un composant, il faut effectuer les actions ci-dessous.

## Analyse statique d'accessibilité

Remplir les champs ci-dessous, en se basant sur l'onglet accessibilité de chaque composant dans le storybook.

![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/9fbd695c-398c-4d5c-9a62-60d29a5e31fa)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 8

En plus mettre une capture de l'onglet en commentaire de cette issue.

## Analyse manuelle

- [x] Est ce que le composant est navigable au clavier (tab) ?
- [x] Est ce que le composant est navigable avec NVDA ?
- [ ] Est ce que le composant est navigable avec un autre lecteur d'écran ?

## Vérification sémantique

- [x] Vérifier l'implémentation sémantique des éléments html ?
- [x] Vérifier la validité HTML W3C (https://validator.w3.org/#validate_by_input) ?
- [x] Vérifier la validité CSS (https://jigsaw.w3.org/css-validator/#validate_by_input) ?

## Problèmes détectés

Si vous avez détectés un défaut d'accessibilité, veuillez créer une issue décrivant le défaut et la rattacher à cette issue.
